docs/userguide/recommenders.md

```diff
@@ -30,17 +30,19 @@ The Bayesian recommenders in BayBE are built on the foundation of the
 class, offering an array of possibilities with internal surrogate models and support
 for various acquisition functions.
 
-* The **[`SequentialGreedyRecommender`](baybe.recommenders.pure.bayesian.sequential_greedy.SequentialGreedyRecommender)**
-  is a powerful recommender that performs sequential Greedy optimization. It can be
-  applied for discrete, continuous and hybrid search spaces. It is an implementation of
-  the BoTorch optimization functions for discrete, continuous and mixed spaces.
-  It is important to note that this recommender performs a brute-force search when
-  applied in hybrid search spaces, as it optimizes the continuous part of the space
-  while exhaustively searching choices in the discrete subspace. You can customize
-  this behavior to only sample a certain percentage of the discrete subspace via the
-  `sample_percentage` attribute and to choose different sampling algorithms via the
-  `hybrid_sampler` attribute. An example on using this recommender in a hybrid space
-  can be found [here](./../../examples/Backtesting/hybrid).
+* The **[`BotorchRecommender`](baybe.recommenders.pure.bayesian.botorch.BotorchRecommender)**
+  is a powerful recommender based on BoTorch's optimization engine that can be applied
+  to all kinds of search spaces. In continuous spaces, its `sequential_continuous` flag
+  allows to chose between greedy sequential optimization and batch optimization as the
+  underlying point generation mode. In discrete/hybrid spaces, sequential greedy
+  selection is the only available mode and is thus activated automatically. Note that
```

Comment:
```suggestion
  selection is the only available mode and is thus activated automatically. 
  
  Note that
```
(suggesting to split the paragraph)

Reply:
Thanks, also added a second split at the end, since the grouping would have been weird otherwise